src/main/java/org/folio/consortia/service/impl/TenantManagerImpl.java

```diff
@@ -87,6 +87,7 @@ public Tenant save(UUID consortiumId, UUID adminUserId, Tenant tenantDto) {
 
     createCustomFieldIfNeeded(tenantDto.getId());
     setUpCentralCustomAuthFlow(tenantDto.getId(), tenantDto.getIsCentral());
+    createIdentityProvider(tenantDto.getId(), new IdentityProviderCreateRequest(true, true));
```

Comment:
need to replace it here instead of adding  one more call
https://github.com/folio-org/mod-consortia-keycloak/blob/master/src/main/java/org/folio/consortia/service/impl/TenantManagerImpl.java#L227